src/sdltiles.cpp

```diff
@@ -1419,8 +1478,13 @@ int HandleDPad()
  */
 long sdl_keysym_to_curses( SDL_Keysym keysym )
 {
-
-    if( get_option<bool>( "DIAG_MOVE_WITH_MODIFIERS" ) ) {
+    std::string arrow_keys_vs_modifiers = get_option<std::string>( "ARROW_KEYS_VS_MODIFIERS" );
+    if ( arrow_keys_vs_modifiers == "numpad" ) {
```

Comment:
Would be nice if you used a consistent code style (here is a space before the `(`, other place don't have it).